backend/benefit/applications/api/v1/views.py

```diff
@@ -313,7 +332,7 @@ def get_queryset(self):
             .prefetch_related(
                 "pay_subsidies", "training_compensations", "calculation__rows"
             )
-        )
+        ).order_by("-handled_at", "-calculation__modified_at")
```

Comment:
The used order_by parameters here and the comment above describing the needed order are out of sync or if they are in sync then please could you explain in more detail how they reflect the same reality. Thank you!

Reply:
The latest version was not pushed to the server, sorry!